tests/sentry/api/endpoints/test_group_details.py

```diff
@@ -78,6 +79,32 @@ def test_pending_delete_pending_merge_excluded(self):
         response = self.client.get(url, format='json')
         assert response.status_code == 404
 
+    def test_environment(self):
```

Comment:
I don't like this test but I felt bad having no significant coverage here

Reply:
Well I for one am glad it's covered.